go.mod

```diff
@@ -0,0 +1,119 @@
+module github.com/influxdata/telegraf
+
+go 1.12
+
+require (
+	cloud.google.com/go v0.37.4
+	code.cloudfoundry.org/clock v0.0.0-20180518195852-02e53af36e6c // indirect
+	collectd.org v0.3.0
+	contrib.go.opencensus.io/exporter/stackdriver v0.6.0 // indirect
+	github.com/Azure/go-autorest v10.12.0+incompatible
+	github.com/Microsoft/ApplicationInsights-Go v0.4.2
+	github.com/Microsoft/go-winio v0.4.9 // indirect
+	github.com/Shopify/sarama v1.19.0
+	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6
+	github.com/aerospike/aerospike-client-go v1.27.0
+	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf
+	github.com/amir/raidman v0.0.0-20170415203553-1ccc43bfb9c9
+	github.com/apache/thrift v0.12.0
+	github.com/aws/aws-sdk-go v1.15.54
+	github.com/bsm/sarama-cluster v2.1.13+incompatible
+	github.com/caio/go-tdigest v2.3.0+incompatible // indirect
+	github.com/cenkalti/backoff v2.0.0+incompatible // indirect
+	github.com/cisco-ie/nx-telemetry-proto v0.0.0-20190531143454-82441e232cf6
+	github.com/couchbase/go-couchbase v0.0.0-20180501122049-16db1f1fe037
+	github.com/couchbase/gomemcached v0.0.0-20180502221210-0da75df14530 // indirect
+	github.com/couchbase/goutils v0.0.0-20180530154633-e865a1461c8a // indirect
+	github.com/denisenkom/go-mssqldb v0.0.0-20190707035753-2be1aa521ff4
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/dimchansky/utfbom v0.0.0-20170328061312-6c6132ff69f0 // indirect
+	github.com/docker/distribution v0.0.0-20170726174610-edc3ab29cdff // indirect
+	github.com/docker/docker v0.0.0-20180327123150-ed7b6428c133
+	github.com/docker/go-connections v0.3.0 // indirect
+	github.com/docker/go-units v0.3.3 // indirect
+	github.com/docker/libnetwork v0.0.0-20181012153825-d7b61745d166
+	github.com/eclipse/paho.mqtt.golang v1.1.1
+	github.com/ericchiang/k8s v1.2.0
+	github.com/ghodss/yaml v0.0.0-20190212211648-25d852aebe32
+	github.com/glinton/ping v0.1.1
+	github.com/go-ini/ini v1.38.1 // indirect
+	github.com/go-logfmt/logfmt v0.4.0
+	github.com/go-redis/redis v6.12.0+incompatible
+	github.com/go-sql-driver/mysql v1.4.0
+	github.com/gobwas/glob v0.2.3
+	github.com/golang/protobuf v1.2.0
+	github.com/google/go-cmp v0.2.0
+	github.com/google/go-github v17.0.0+incompatible
+	github.com/google/go-github/v24 v24.0.1 // indirect
+	github.com/google/uuid v0.0.0-20161128191214-064e2069ce9c // indirect
+	github.com/googleapis/gax-go v2.0.0+incompatible // indirect
+	github.com/gorilla/mux v1.6.2
+	github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed // indirect
+	github.com/harlow/kinesis-consumer v0.0.0-20181230152818-2f58b136fee0
+	github.com/hashicorp/consul v1.2.1
+	github.com/hashicorp/go-cleanhttp v0.0.0-20171218145408-d5fe4b57a186 // indirect
+	github.com/hashicorp/go-rootcerts v0.0.0-20160503143440-6bb64b370b90 // indirect
+	github.com/hashicorp/serf v0.8.1 // indirect
+	github.com/influxdata/go-syslog v1.0.1 // indirect
+	github.com/influxdata/go-syslog/v2 v2.0.0
```

Comment:
Why is go-syslog listed twice?  Is this just one of the `go mod tidy` things?

Reply:
🤔 I will investigate. Looks like it is keeping both versions around. Maybe we have references to both.

Reply:
Turns out I needed to do a `go mod tidy`!